libnavigation-base/src/main/java/com/mapbox/navigation/base/options/NavigationOptions.kt

```diff
@@ -42,6 +42,7 @@ const val DEFAULT_NAVIGATOR_PREDICTION_MILLIS = 1000L
  * @param historyRecorderOptions defines configuration for recording navigation events
  * @param eventsAppMetadata [EventsAppMetadata] information (optional)
  * @param enableSensors enables sensors for current position calculation (optional)
+ * @param copilotOptions defines options for Copilot
  */
 class NavigationOptions private constructor(
```

Comment:
```suggestion
class NavigationOptions @OptIn(ExperimentalPreviewMapboxNavigationAPI::class) private constructor(
```
Also `toBuilder` and `toString` have to be marked with `@OptIn(ExperimentalPreviewMapboxNavigationAPI::class)`.